Resources/doc/reference/translation.rst

```diff
@@ -49,10 +49,10 @@ translate messages within the ``configureFields`` method or in templates.
     {{ 'message_create_snapshots'|trans({}, 'SonataPageBundle') }}
 
     {# by using the admin trans method with hardcoded catalogue #}
-    {{ admin.trans('message_create_snapshots', {}, 'SonataPageBundle') }}
+    {{ 'message_create_snapshots'|trans({}, 'SonataPageBundle') }}
 
     {# by using the admin trans with the configured catalogue #}
-    {{ admin.trans('message_create_snapshots') }}
+    {{ 'message_create_snapshots'|trans({}, admin.translationdomain) }}
```

Comment:
> Isn't it enabled by default?

[doesn't look like it is](https://github.com/symfony/symfony-standard/blob/master/app/config/config.yml#L13)

> We have some others parts where we already used the trans filter.

Then maybe we should require the translator to be enabled explicitly and deprecate "not using it". That way we could drop that piece of code.